### ISSUE TITLE
Online documentation does not cover new bracket notation for room objects

### ISSUE THREAD

**Issue by no1453** [JasonLautzenheiser/trizbort#478]
The new tags like "[u]supporter" or "[p]Neutral Person", etc., seem like a great innovation,, but I'm having trouble using them because the online documentation doesn't mention them or their use. (I'm trying to make maps to export to ZIL format, and the results of my attempts to guess how the brackets work has been unfruitful.)
![Capture](https://user-images.githubusercontent.com/3926817/59542516-8a34e500-8ec3-11e9-9d07-591fac395207.PNG)

**Comment by no1453** [comment]
Yes, I was an Infocom fan in the '80s, the source release was like a godsend.   I was quite surprised to find my favorite mapper supported ZIL!     Thanks for that.

**Comment by JasonLautzenheiser** [comment]
When Jesse McGrew first starting putting out his ZIL compiler, I figured it would be a good opportunity to put support for it in Trizbort.  He added some enhancements to it as well and we have at least what we have.  I'm sure I could go back and add some more enhancements to it now.
